windows-calico/quickstart.md

```diff
@@ -321,6 +330,69 @@ The following steps install a Kubernetes cluster on a single Windows node, with
    ```
 %>
 
+<label:AKS>
+<%
+
+1. Register `EnableAKSWindowsCalico` feature flag with the following azure cli commad.
+
+   ```bash
+   az feature register --namespace "Microsoft.ContainerService" --name "EnableAKSWindowsCalico"
+   ```
+
+1. Wait until `EnableAKSWindowsCalico` feature flag is registered successfully. Execute following cli command to get current status of the feature.
+
+   ```bash
+   az feature list -o table --query "[?contains(name, 'Microsoft.ContainerService/EnableAKSWindowsCalico')].{Name:name,State:properties.state}"
+   ```
+   
+   Move to next step if you see the same output from above command.
```

Comment:
Move to next step if the output from above command matches the following output.